include/mbgl/map/map_options.hpp

```diff
@@ -131,6 +131,22 @@ class MapOptions {
      */
     bool crossSourceCollisions() const;
 
+    /**
+     * @brief Sets the local font family, which is the local font family name for client-side
+     * rendering of CJK glyphs. Default font for local ideograph font family is "PingFang".
```

Comment:
"PingFang" is only used as default on `iOS`. The feature is not enabled by default on core because it is platform specific.